src/app/pages/applications/chart-releases/chart-releases.component.ts

```diff
@@ -217,16 +217,27 @@ export class ChartReleasesComponent implements AfterViewInit, OnInit, OnDestroy
             this.chartItems.clear();
             this.showLoadStatus(EmptyType.Errors);
           } else {
-            this.appService.getChartReleases().pipe(untilDestroyed(this)).subscribe((charts) => {
-              this.chartItems.clear();
+            try {
+              this.appService.getChartReleases().pipe(
+                catchError(() => of(this.showLoadStatus(EmptyType.Errors))),
+                untilDestroyed(this),
+              ).subscribe((charts) => {
+                if (!charts) {
+                  return this.showLoadStatus(EmptyType.Errors);
+                }
+
+                this.chartItems.clear();
+
+                charts.forEach((chart) => {
+                  chart.selected = false;
+                  this.chartItems.set(chart.name, chart);
+                });
 
-              charts.forEach((chart) => {
-                chart.selected = false;
-                this.chartItems.set(chart.name, chart);
+                this.filterChartItems();
               });
-
-              this.filterChartItems();
-            });
+            } catch {
```

Comment:
Try/catch looks redundant since we're catching the error now.

Reply:
yeah, I put it to catch manual **throw new Error()**, will remove